includes/helpers.inc

```diff
@@ -58,66 +57,6 @@ function _tide_read_config($id, array $locations = [], $override = FALSE) {
   return $storages[$path]->read($id);
 }
 
-/**
```

Comment:
I assume this wasn't being used and is safe to remove @sonnykt. Is there another reason it is being removed? Would it actually work as deleting the file is probably not enough as the next config export will reinstate the files?

Reply:
@anthony-malkoun it was added in the previous implementation of SDPA-309, which was not correct. It only works on env having write permission to config files - aka local only. It has never worked on lagoon as the config files are managed by git deployment.

Reply:
That makes sense to me @sonnykt. Thanks for explaining.